ModelViewer/ModelViewer/ViewerGraphicsWindow.cpp

```diff
@@ -125,6 +132,41 @@ void ViewerGraphicsWindow::wheelEvent(QWheelEvent* event)
     }
 }
 
+void ViewerGraphicsWindow::keyPressEvent(QKeyEvent* event)
+{
+    if (event->type() == QEvent::KeyPress) {
+        keys += ((QKeyEvent*)event)->key();
+    }
+
+    QWindow::keyPressEvent(event);
+}
+
+void ViewerGraphicsWindow::keyReleaseEvent(QKeyEvent* event)
+{
+    if (event->type() == QEvent::KeyRelease) {
+        if (keys.contains(Qt::Key_Control) && keys.contains(Qt::Key_S))
```

Comment:
There's an easier way to add shortcuts.  I'd recommend using this method because the shortcut will be displayed in the menu bar.  For example:
![image](https://user-images.githubusercontent.com/13424385/98192135-52bb4400-1ecf-11eb-82b7-5c3449e84446.png)

The AddAction function has an override which takes a QKeySequence as a shortcut.  Here's an example:
https://doc.qt.io/qt-5/qmenu.html#addAction-4
ModelViewer.cpp
`pViewMenu->addAction("Reset", [=] { m_pGraphicsWindow->resetView(); }, QKeySequence(Qt::CTRL + Qt::Key_R));`